src/tools/auth0/handlers/hooks.ts

```diff
@@ -234,15 +242,29 @@ export default class HooksHandler extends DefaultHandler {
     // Do nothing if not set
     if (!hooks) return;
 
-    // Figure out what needs to be updated vs created
-    const changes = await this.calcChanges(assets);
-    await super.processChanges(assets, {
-      del: changes.del,
-      create: changes.create,
-      update: changes.update,
-      conflicts: changes.conflicts,
-    });
+    log.warn(
+      'Hooks are deprecated and should be migrated to actions instead. See: https://auth0.com/docs/customize/actions/migrate/migrate-from-hooks-to-actions for more information.'
```

Comment:
```suggestion
      'Hooks are deprecated, migrate to using actions instead. See: https://auth0.com/docs/customize/actions/migrate/migrate-from-hooks-to-actions for more information.'
```

Just a suggestion. Maybe this reads better? 🤔